File: julia/wordcount.jl
counts = Dict{AbstractString, UInt64}()

for line in eachline(stdin)
  for m in eachmatch(r"[a-z\']+", lowercase(line))
    counts[m.match] = get(counts, m.match, 0) + 1
  end
end
for (word, count) in sort(collect(counts))
  println("$word $count")
end

File: python/wordcount.py
import sys, re

counts = {}
for line in sys.stdin:
    for word in re.findall(r'[a-z\']+', line.lower()):
        counts[word] = counts.get(word, 0) + 1

for word, count in sorted(counts.items()):
    print(word, count)
